test/parallel/test-module-setsourcemapssupport.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+require('../common');
+const assert = require('node:assert');
+const Module = require('node:module');
+
+{
```

Comment:
Nit: comments here explaining what is being tested for would be helpful.

Reply:
Thanks, updated!